templates/legal/ubuntu-pro-service-terms/_base_uast_markdown.html

```diff
@@ -16,6 +16,7 @@
             <div class="p-card">
               <h3>Older versions</h3>
               <ul class="p-list">
+                <li class="p-list__item"><a href="/legal/ubuntu-pro-service-terms/2024-12-09">09 December 2024&nbsp;&rsaquo;</a></li>
```

Comment:
@andesol Can you please check from the stakeholder if adding a new older version is needed in this case, since there is only one new word added to the content ("tariffs").

Reply:
Also, even if you add a new link to an older version, it needs to be a date from the past, when the previous version was released, not the current date.

Reply:
Confirmed (comment in the copy doc): we have to keep the older version even if it's one line.

From what I see in the commit history, the last modification on this file was on Feb 16. Should we consider that the release date? If so, the update is done, ready for a review.